diff --git a/node_modules/openclaw/dist/server-Cv5hzFG4.js b/node_modules/openclaw/dist/server-Cv5hzFG4.js
index 926102f391c94b0e8bb52c2d322c107636e267f3..9e04644fcb8cc406d9b50f2590c5e12788cb7b6e 100644
--- a/node_modules/openclaw/dist/server-Cv5hzFG4.js
+++ b/node_modules/openclaw/dist/server-Cv5hzFG4.js
@@ -26710,7 +26710,7 @@
 					close(1008, truncateCloseReason(authMessage));
 				};
 				const clearUnboundScopes = () => {
-					if (scopes.length > 0) {
+					if (scopes.length > 0 && !sharedAuthOk) {
 						scopes = [];
 						connectParams.scopes = scopes;
 					}
